### DISCUSSION TITLE
Lock task mode support on Android

### DISCUSSION THREAD

**Discussion by ThorANilsson** [expo/expo#31872] [Proposals and Ideas]
Android has a great built-in way to turn the device into a kiosk, which only allows one or a set of apps be to be used. Great for self-service scenarios like checking in to a hotel or similar, where you don't want the user to exit the app. This is called Lock task mode.

More info about "Lock task mode" here => https://developer.android.com/work/dpc/dedicated-devices/lock-task-mode

I weren't able to find any examples of this being done using Expo apps, but I think it would be a great (and needed!) feature.

**Comment by akbaraditamasp** [comment]
Try my package https://www.npmjs.com/package/@akbaraditamasp/expo-lock-task

**Reply by vehbiu** [reply]
@akbaraditamasp Thanks!
